src/wfcopy.c

```diff
@@ -1685,7 +1691,14 @@ GetNextPair(PCOPYROOT pcr, LPTSTR pFrom,
                   goto ReturnPair;
                }
 
-               //
+               // Skip reparse points
+               if (dwFunc == FUNC_DELETE && pDTA->fd.dwFileAttributes & (ATTR_SYMBOLIC | ATTR_JUNCTION)) {

```

Comment:
"Skip" doesn't seem like the right point; we are returning the reparse point as a directory, but not recursing.

Reply:
A better comment would be 
'Return reparse point and delete it via OPER_RMDIR'